pyproject.toml

```diff
@@ -5,8 +5,8 @@ requires = [
     "setuptools",
     "wheel",
     "Cython>=0.29.16",  # Note: sync with setup.py
-    "numpy==1.13.3; python_version=='3.6' and platform_system!='AIX'",
-    "numpy==1.14.5; python_version>='3.7' and platform_system!='AIX'",
+    "numpy==1.15.4; python_version=='3.6' and platform_system!='AIX'",
+    "numpy==1.15.4; python_version>='3.7' and platform_system!='AIX'",
```

Comment:
Shouldn't that be "numpy>=1.15.4"? I had issues installing pandas from source with numpy 1.18.5 installed.
Should I create a ticket for that?

Reply:
I don't believe so. We build with the oldest supported NumPy that is compatible with the version of NumPy.

> I had issues installing pandas from source with numpy 1.18.5 installed.

This also doesn't quite make sense. The idea behind pyproject.toml is to isolate the build environment. It shouldn't matter what version of NumPy was used to build pandas with, as long as it's binary compatible (IIUC).

Reply:
Ok, ignore my comment if that is how you make sure that your build with the oldest supported version.

In my case I have numpy 1.18.5 installed and I want to use it to build pandas from source. However, it complains that the build requirements are note met. 
```
Installing build dependencies ... error
ERROR: Complete output from command /opt/project/venv37/bin/python3.7 /opt/project/venv37/lib/python3.7/site-packages/pip install --ignore-installed --no-user --prefix /tmp/pip-build-env-0xz070yu/overlay --no-warn-script-location --no-binary :none: --only-binary :none: --no-index --find-links https://project:omniscale@pkgs.omniscale.net:443/project -- setuptools wheel 'Cython>=0.29.13' 'numpy==1.13.3; python_version=='"'"'3.6'"'"' and platform_system!='"'"'AIX'"'"'' 'numpy==1.14.5; python_version>='"'"'3.7'"'"' and platform_system!='"'"'AIX'"'"'' 'numpy==1.16.0; python_version=='"'"'3.6'"'"' and platform_system=='"'"'AIX'"'"'' 'numpy==1.16.0; python_version>='"'"'3.7'"'"' and platform_system=='"'"'AIX'"'"'':
ERROR: Ignoring numpy: markers 'python_version == "3.6" and platform_system != "AIX"' don't match your environment
Ignoring numpy: markers 'python_version == "3.6" and platform_system == "AIX"' don't match your environment
Ignoring numpy: markers 'python_version >= "3.7" and platform_system == "AIX"' don't match your environment
Looking in links: https://*redacted*
Collecting setuptools
Collecting wheel
Collecting Cython>=0.29.13
Collecting numpy==1.14.5
  ERROR: Could not find a version that satisfies the requirement numpy==1.14.5 (from versions: 1.18.5)
ERROR: No matching distribution found for numpy==1.14.5
```

Reply:
However, were you using python 3.8? There was an issue with us not having a line for py38. https://github.com/pandas-dev/pandas/pull/34667

Reply:
No, 3.7.

Reply:
Ok, I think I didn't quite understood how build isolation works. Thanks for your quick relies!